### PR TITLE
Fixed wscript files so that waf can be used to compile opfor.

### PR DESCRIPTION
**Pull request by sedme0**

### CODE REVIEW

File: dlls/wscript [hunk 1]
```diff
@@ -27,6 +27,21 @@ def build(bld):
 	])
 
 	source += [
+		'gearbox/blkop_apache.cpp', 'gearbox/blkop_osprey.cpp', 'gearbox/cleansuit_scientist.cpp',
```
Comment (a1batross) [inline dlls/wscript:30]: Check if you can do bld.path.ant_glob(gearbox/*.cpp) instead.

I didn't used glob here because some files are just sample code.

**Comment by h1naria** [comment]
Need to merge changes from master at first.

File: wscript [hunk 2]
```diff
@@ -184,7 +184,7 @@ def configure(conf):
 	compiler_optional_flags = [
 		'-fdiagnostics-color=always',
 		'-Werror=return-type',
-		'-Werror=parentheses',
+		#'-Werror=parentheses',
```
Comment (h1naria) [inline wscript:187]: Didn't comment this without reason.

**Comment by FreeSlave** [comment]
Btw in cmake I made ropes.cpp compiling with -O0 (without optimizations). With optimizations it has a weird glitch when a player can randomly teleport between barnacle and rope if both are present on the map. So either we should fix the issue or make the same hack for waf.

**Comment by nekonomicon** [comment]
@FreeSlave I didn't understand why memset was used in this code, and there another code style.

**Comment by FreeSlave** [comment]
I guess it was fixed in https://github.com/FWGS/hlsdk-xash3d/commit/805fce2350179d1f11c8ff65232571d866fb3072
Can we close this PR?

**Comment by a1batross** [comment]
Ye, right.

**Pull request closed without merging**